<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Histomerge</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="videoLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>167000</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="recordButton">
         <property name="maximumSize">
          <size>
           <width>16000</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="accessibleName">
          <string>Grabar</string>
         </property>
         <property name="text">
          <string>Grabar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16000</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Detener</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="captureButton">
         <property name="maximumSize">
          <size>
           <width>16000</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Capturar
Frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="maximumSize">
          <size>
           <width>16000</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Salir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionSeleccionar_carpeta_de_guardado"/>
    <addaction name="actionSeleccionar_camara"/>
    <addaction name="separator"/>
    <addaction name="actionFormato_de_salida"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAtajos_de_teclado"/>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuOpciones"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSeleccionar_carpeta_de_guardado">
   <property name="text">
    <string>Seleccionar carpeta de guardado</string>
   </property>
  </action>
  <action name="actionSeleccionar_camara">
   <property name="text">
    <string>Seleccionar c√°mara</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionAtajos_de_teclado">
   <property name="text">
    <string>Atajos de teclado</string>
   </property>
  </action>
  <action name="actionFormato_de_salida">
   <property name="text">
    <string>Formato de salida</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
